mysql/oressource.sql

```diff
@@ -34,9 +34,9 @@ CREATE TABLE IF NOT EXISTS `collectes` (
   `localisation` int(11) NOT NULL,
   `id_point_collecte` int(11) NOT NULL,
   `commentaire` text NOT NULL,
-  `id_createur` int(11) NOT NULL,
-  `id_last_hero` int(11) NOT NULL,
-  `last_hero_timestamp` timestamp NOT NULL DEFAULT '0000-00-00 00:00:00',
+  `id_createur` int(11) NOT NULL DEFAULT 0,
+  `id_last_hero` int(11) NOT NULL DEFAULT 0,
+  `last_hero_timestamp` timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
```

Comment:
Malheureusement, les valeurs '0000-00-00 00:00:00' ET 0 ne sont pas acceptées par MySql : 

Erreur dans la requête (1067): Invalid default value for 'last_hero_timestamp' 

Je me suis résigné à mettre CURRENT_TIMESTAMP (que je préfère aussi).